<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductManagerForm</class>
 <widget class="QWidget" name="ProductManagerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client Info</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QToolBox" name="toolBox">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="inputPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>284</width>
         <height>366</height>
        </rect>
       </property>
       <attribute name="label">
        <string>&amp;Input</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QFormLayout" name="clientFormLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="idLabel">
              <property name="text">
               <string>&amp;PID</string>
              </property>
              <property name="buddy">
               <cstring>idLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="idLineEdit">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="pnameLabel">
              <property name="text">
               <string>&amp;ProductName</string>
              </property>
              <property name="buddy">
               <cstring>pnameLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="pnameLineEdit"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="priceLabel">
              <property name="text">
               <string>&amp;Price</string>
              </property>
              <property name="buddy">
               <cstring>priceLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="priceLineEdit"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="stockLabel">
              <property name="text">
               <string>&amp;Stock</string>
              </property>
              <property name="buddy">
               <cstring>stockLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="stockLineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>14</width>
              <height>14</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="addPushButton">
              <property name="text">
               <string>&amp;Add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="modifyPushButton">
              <property name="text">
               <string>&amp;Modify</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="deletePushButton">
              <property name="text">
               <string>&amp;Delete</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="searchPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>284</width>
         <height>366</height>
        </rect>
       </property>
       <attribute name="label">
        <string>&amp;Search</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTreeWidget" name="searchTreeWidget">
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <column>
           <property name="text">
            <string>PID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ProductName</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Price</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Stock</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="searchComboBox">
            <item>
             <property name="text">
              <string>PID</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ProductName</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Price</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Stock</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="searchLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="searchPushButton">
          <property name="text">
           <string>S&amp;earch</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QTreeWidget" name="producttreeWidget">
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <column>
       <property name="text">
        <string>PID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ProductName</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Stock</string>
       </property>
      </column>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
